src/panels/config/devices/ha-config-device-page.ts

```diff
@@ -798,6 +800,274 @@ export class HaConfigDevicePage extends LitElement {
             )}
           </div>
           <div class="column">
+          ${
+            this.narrow && isComponentLoaded(this.hass, "automation")
```

Comment:
To prevent so much duplicate code, can we store the `html` in a const and place it where we want based on narrow or not?